extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesConfig.java

```diff
@@ -193,11 +199,12 @@
 
     /**
      * The target deployment platform.
-     * Defaults to kubernetes. Can be kubernetes, openshift, knative etc, or any combination of the above as comma separated
+     * Defaults to kubernetes. Can be kubernetes, openshift, knative, minikube etc, or any combination of the above as comma
+     * separated
      * list.
      */
-    @ConfigItem(defaultValue = "kubernetes")
-    List<String> deploymentTarget;
+    @ConfigItem
+    Optional<List<String>> deploymentTarget;
```

Comment:
I guess that `Optional` is introduced in order to be able to easily tell when this is a default value or a user provided one?

Reply:
Yes, exactly